JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -72,18 +73,18 @@ fun ArticleScreen(postId: String) {
                 title = {
                     Text(
                         text = "Published in: ${post.publication?.name}",
-                        style = MaterialTheme.typography().subtitle2
+                        style = MaterialTheme.typography.subtitle2.copy(color = contentColor())
```

Comment:
Shouldn't `contentColor()` be the default? Is this needed?

Reply:
It was rastering as black text on a dark red background. Will file bug and add FIXME pointing to it since I agree this should not be needed.

Reply:
added